labrad/node/server_config.py

```diff
@@ -0,0 +1,114 @@
+import io
+import os
+import sys
+
+from configparser import ConfigParser
+
+from labrad.util import findEnvironmentVars
+
+
+class ServerConfig(object):
```

Comment:
Currently the node server modifies the name of some server configs to handle the case where we detect multiple versions of a given server (https://github.com/labrad/pylabrad/blob/master/labrad/node/__init__.py#L749). Unfortunately that makes converting this a tuple more difficult, so I'm going to leave it as-is for now. I'm working on refactoring the version handling code, and then could make that change, because it would be nice for this to be immutable.